addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -980,8 +970,9 @@ public void OnPluginStart()
 	BuildPath(Path_SM, eLog, sizeof(eLog), "%s/%s", LogPath, ErrorLog);
 	if(!FileExists(eLog))
 		OpenFile(eLog, "a+");
-	
-	g_FF2Protected = new FF2Protected();
+		
+	if(!g_FF2Protected)

```

Comment:
You don't need to check if Handle is null in `OnPluginStart` because this forward calls only on plugin start/reload.
So this Handle will always be `null` before `OnPluginStart`.

Reply:
a safe check in case the handle was already created, i cant be sure 100% without a propper lookup about whats firing first. can be OnMapStart or OnPluginStart, depends on lateload/map change

Reply:
Yeah, but this check is not needed in `OnPluginStart` just because Handle will be **null** at this point.

Reply:
So you can safely remove this line, just leave `g_FF2Protected = new FF2Protected();` in `OnPluginStart`.

Reply:
I'm going to take a precaution here, its unpredicted to assume OnMapStart start bedore OnPluginStart, it can only if the plugin wasnt reloaded or lateloaded.

Reply:
As i remember, `OnMapStart` fires after `OnPluginStart` even if you reload the plugin. You can try it yourself.

Reply:
alright, should be good enough